File: array/CheckPrime.java
import java.util.*;
public class CheckPrime
{
	public static void main(String x[])
	{
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter a number :");
		int num=sc.nextInt();
		int count=0;
		for(int i=1;i<=num;i++)
		{
			if(num%i==0)
			{
				count++;
			}
		}
		if(count==2)
		{
			System.out.printf("Prime Number");	
		}
		else
		{
			System.out.printf("Not prime number");
		}
	}
}

File: function/FUNCTION RECURSION/checkPrime.c
#include <stdio.h>
int i = 1,count=0;
int main(int argc, char const *argv[])
{
    int checkPrime(int); // declaration
    int num;
    printf("enter number : ");
      scanf("%d", &num);
     checkPrime(num);
    return 0;
}
int checkPrime(int n)
{
    if (i <= n)
    { 
       if(n%i==0)
       {
        count++;
         
       }
        i++;
        return checkPrime(n);
    }
    //  printf(" %d",count);  
     if (count==2)
     {
        printf("number is prime number ");
     }
     else
       printf("number is not prime number ");

}